### PR TITLE
VMHostSettings & vCenterSettings Integration tests

### PR DESCRIPTION
**Pull request by lucdekens**
New attempt to get these tests merged.
The files are now formatted with the workspace settings, so the diffs should be better.

### CODE REVIEW

File: Source/VMware.vSphereDSC/Tests/Integration/Configurations/VMHostSettings/VMHostSettings_Config.ps1 [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+$Password = $Password | ConvertTo-SecureString -AsPlainText -Force
+$script:vmHostCredential = New-Object System.Management.Automation.PSCredential($User, $Password)
+
+$script:motd = 'VMHostSettings motd test'
+$script:issue = 'VMHostSettings issue test'
+
+$script:configurationData = @{
+    AllNodes = @(
+        @{
+            NodeName                    = 'localhost'
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/Configurations/VMHostSettings/VMHostSettings_Config.ps1]: Fix the spacing here between NodeName and the equal sign.

File: Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1 [hunk 2]
```diff
@@ -40,7 +40,7 @@ $script:taskMaxAge = 40
 $script:configurationData = @{
     AllNodes = @(
         @{
-            NodeName = 'localhost'
+            NodeName                    = 'localhost'
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1]: Same as the above one.

File: Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1 [hunk 3]
```diff
@@ -53,10 +53,9 @@ Configuration vCenterSettings_WithLoggingLevel_Config {
     Import-DscResource -ModuleName VMware.vSphereDSC
 
     Node localhost {
-        vCenterSettings vCenterSettings
-        {
-            Server = $Server
-            Credential = $script:vCenterCredential
+        vCenterSettings vCenterSettings {
+            Server       = $Server
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1]: Same as the above one.

File: Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1 [hunk 4]
```diff
@@ -66,12 +65,11 @@ Configuration vCenterSettings_WithEventMaxAge_Config {
     Import-DscResource -ModuleName VMware.vSphereDSC
 
     Node localhost {
-        vCenterSettings vCenterSettings
-        {
-            Server = $Server
-            Credential = $script:vCenterCredential
+        vCenterSettings vCenterSettings {
+            Server             = $Server
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1]: The same for the spacing below.

File: Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1 [hunk 5]
```diff
@@ -80,16 +78,34 @@ Configuration vCenterSettings_WithTaskMaxAge_Config {
...
+        vCenterSettings vCenterSettings {
+            Server            = $Server
+            Credential        = $script:vCenterCredential
+            TaskMaxAgeEnabled = $script:taskMaxAgeEnabled
+            TaskMaxAge        = $script:taskMaxAge
+        }
+    }
+}
+
+vCenterSettings_WithLoggingLevel_Config -OutputPath "$integrationTestsFolderPath\vCenterSettings_WithLoggingLevel_Config" -ConfigurationData $script:configurationData
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1]: Why do we have duplicate lines as those at the end of the file ?

File: Source/VMware.vSphereDSC/Tests/Integration/vCenterSettings.Integration.Tests.ps1 [hunk 6]
```diff
@@ -45,35 +46,46 @@ $script:currentEventMaxAgeEnabled = $null
...
 $script:eventMaxAgeEnabled = $false
 $script:eventMaxAge = 40
 $script:taskMaxAgeEnabled = $false
 $script:taskMaxAge = 40
+$script:motd = 'vCenterSettings motd test'
+$script:issue = 'vCenterSettings issue test'
 
 $script:resourceWithLoggingLevel = @{
-    Server = $Server
+    Server       = $Server
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/vCenterSettings.Integration.Tests.ps1]: Fix all the spacing problems in the file.

File: Source/VMware.vSphereDSC/Tests/Integration/vCenterSettings.Integration.Tests.ps1 [hunk 7]
```diff
@@ -237,6 +257,60 @@ Describe "$($script:dscResourceName)_Integration" {
...
+            { $script:dscConfigWithTaskMaxAge } | Should -Not -Throw
+
+            $configuration.Server | Should -Be $script:resourceWithTaskMaxAge.Server
+            $configuration.LoggingLevel | Should -Be ($script:currentLoggingLevel).Value
+            $configuration.EventMaxAgeEnabled | Should -Be ($script:currentEventMaxAgeEnabled).Value
+            $configuration.EventMaxAge | Should -Be ($script:currentEventMaxAge).Value
+            $configuration.TaskMaxAgeEnabled | Should -Be ($script:currentTaskMaxAgeEnabled).Value
+            $configuration.TaskMaxAge | Should -Be ($script:currentTaskMaxAge).Value
+            $configuration.Motd | Should -Be $script:resourceWithMotd.Motd
+            $configuration.Issue | Should -Be $script:resourceWithIssue.Issue
```
Comment (SimeonGerginov) [inline Source/VMware.vSphereDSC/Tests/Integration/vCenterSettings.Integration.Tests.ps1]: Remove empty line here.

**Comment by lucdekens** [comment]
Updated files, and pushed changes

**Pull request merged**